packages/contracts-bedrock/.storage-layout

```diff
@@ -6,25 +6,25 @@
 ➡ contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger
 =======================
 
-| Name               | Type                     | Slot | Offset | Bytes | Contract                                                       |
-|--------------------|--------------------------|------|--------|-------|----------------------------------------------------------------|
-| spacer_0_0_20      | address                  | 0    | 0      | 20    | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| _initialized       | uint8                    | 0    | 20     | 1     | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| _initializing      | bool                     | 0    | 21     | 1     | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| __gap              | uint256[50]              | 1    | 0      | 1600  | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| _owner             | address                  | 51   | 0      | 20    | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| __gap              | uint256[49]              | 52   | 0      | 1568  | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| _paused            | bool                     | 101  | 0      | 1     | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| __gap              | uint256[49]              | 102  | 0      | 1568  | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
-| _status            | uint256                  | 151  | 0      | 32    | contracts/L1/L1CrossDomainMessenger.sol:L1CrossDomainMessenger |
```

Comment:
Do you mind adding a specific spacer for `_status`? Its technically semantically different to have a `__gap[50]` vs a `__gap[49]` + `_status` because the gap indicates that it was never used before. I think status will be set to 1 meaning that we will have dirty storage in a gap which would break an assumption

 It should also make the diff in this file easier to read